packages/launchpad/src/signingcosmosclient.spec.ts

```diff
@@ -172,4 +179,109 @@ describe("SigningCosmosClient", () => {
       assertIsBroadcastTxSuccess(result);
     });
   });
+
+  describe("sign", () => {
+    it("works", async () => {
+      pendingWithoutLaunchpad();
+      const wallet = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic);
+      const client = new SigningCosmosClient(launchpad.endpoint, faucet.address0, wallet);
+
+      const msg1: MsgDelegate = {
+        type: "cosmos-sdk/MsgDelegate",
+        value: {
+          delegator_address: faucet.address0,
+          validator_address: launchpad.validator.address,
+          amount: coin(1234, "ustake"),
+        },
+      };
+      const msg2: MsgSend = {
+        type: "cosmos-sdk/MsgSend",
+        value: {
+          from_address: faucet.address0,
+          to_address: makeRandomAddress(),
+          amount: coins(1234567, "ucosm"),
+        },
+      };
+      const fee = {
+        amount: coins(2000, "ucosm"),
+        gas: "180000", // 180k
+      };
+      const memo = "Use your power wisely";
+
+      const signed = await client.sign([msg1, msg2], fee, memo);
+      expect(signed.msg).toEqual([msg1, msg2]);
+      expect(signed.fee).toEqual(fee);
+      expect(signed.memo).toEqual(memo);
+      expect(signed.signatures).toEqual([
```

Comment:
Nice, just assert there is a sig here. And the broadcastResult asserts this is formatted correctly.